backend/src/v2/driver/resolve.go

```diff
@@ -193,7 +194,12 @@ func resolveInputs(
 				}
 				switch v := value.GetKind().(type) {
 				case *structpb.Value_NullValue:
-					return fmt.Errorf("got null for input parameter %q", name)
+					// If isDagDriver is true, allow input parameter to have null value.
+					if isDagDriver {
+						return nil
```

Comment:
We still need to validate the other inputs.
```suggestion
						continue
```

Reply:
Updated